charts/dify/values.yaml

```diff
@@ -241,6 +241,8 @@ proxy:
   customReadinessProbe: {}
   ## @param proxy.customStartupProbe Custom startupProbe that overrides the default one
   customStartupProbe: {}
+  ## @param.clientMaxBodySize Custom client_max_body_size param nginx default: 15m
```

Comment:
keyword `proxy` was missing. Do you mean
`## @param proxy.clientMaxBodySize Custom client_max_body_size param nginx default: 15m`